Makefile

```diff
@@ -73,7 +73,11 @@ livedocs: reqs
 package:
 	rm -rf $(PKGDISTDIR)
 	rm -rf $(PKGBUILDDIR)
-	$(PY3) setup.py sdist bdist_wheel
+	$(VENV_DIR)/bin/python setup.py sdist bdist_wheel
+
+.PHONY: publish
+publish: package
+	$(VENV_DIR)/bin/python -m twine upload dist/*
```

Comment:
Little nervous that this may try to upload out-of-date versions and therefore error out before uploading the most recent version, but if this works, then :+1:

Reply:
What do you mean out-of-date versions?

Reply:
So if you run `make package`, it will build a source distribution and binary wheel distribution in `dist/`:

```bash
$ ls dist/
orquesta-1.1.tgz
orquesta-1.1.whl
```
(making up filenames here)

So then you run `make publish`, and it uploads `orquesta-1.1.tgz` and `orquesta-1.1.whl` to PyPI - :+1: all is good.

Development continues, and you have version `0.2` ready, so you run `make package` again, and it adds those to `dist/`:

```bash
$ ls dist/
orquesta-1.1.tgz
orquesta-1.1.whl
orquesta-1.2.tgz
orquesta-1.2.whl
```

And then when you go to `make publish` the package, `twine` tries to upload all of the files in `dist/*`.

But PyPI already has a `1.1` version released, and will reject any further uploads for the same version.

So if twine tries to upload `orquesta-1.1.tgz` or `orquesta-1.1.whl` before `orquesta-1.2.tgz` or `orquesta-1.2.whl`, PyPI will reject it and twine might fail.

A workaround would be to `rm` everything in `dist/*` in the `clean` target. Then you could `make clean publish` and it would only try to upload the most recent version.

If you wanted a better workaround, you could only try to publish the last created/modified `.whl` or `.tgz` (or whatever) files.

Reply:
Good catch. I'll add the clean command during package.

Reply:
Actually, I already remove those directories before I run the package command.